src/Orchard/Tasks/Locking/Services/Lock.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Orchard.Logging;
+
+namespace Orchard.Tasks.Locking.Services {
+
+    /// <summary>
+    /// Provides a database driven implementation of <see cref="IDistributedLock" />
+    /// </summary>
+    public class Lock : IDistributedLock {
```

Comment:
We should get rid of this interface, and call the class `DistributedLock` I think.